queue.h

```diff
@@ -57,12 +57,9 @@ queue_empty(queue_t *);
  * Add the given item to the given queue.
  *
  * This call is synchronized and thus safe to call from different threads.
- *
- * This call will block if the given queue is full. It will only return once the
- * given item could successfully be added to the queue.
```

Comment:
This comment removal should go in the commit removing the `not_full` condition variable.

Reply:
Moved.